arangod/Aql/grammar.y

```diff
@@ -730,6 +732,15 @@ k_paths_graph_info:
     }
   ;
 
+all_shortest_paths_graph_info:
+    graph_direction T_ALL_SHORTEST_PATHS expression T_STRING expression graph_subject options {
+      auto nodeStart = parser->ast()->createNodeValueInt(1);
```

Comment:
Is this actually correct?

What about the pathological case of ALL_SHORTEST_PATH "A" TO "A" ?
This should be one, and that should have an empty list of edges, and only A as a vertex.

Reply:
good catch!

With `auto nodeStart = parser->ast()->createNodeValueInt(1);`, running `ALL_SHORTEST_PATHS "A" TO "A"` returns return an empty list `[]`.

With `auto nodeStart = parser->ast()->createNodeValueInt(0);`, running `ALL_SHORTEST_PATHS "A" TO "A"` returns an empty list of edges, and only A as a vertex.

This has been addressed in 7dd39c35d94